### DISCUSSION TITLE
is it possible to change the Contributor name of someone ?

### DISCUSSION THREAD

**Discussion by borisBelloc** [community/community#22279] [New to GitHub]
<p>Hello,</p>
<p>I tried to find a solution to my problem and didnt find any (english isnt my primary language so it didnt help :confused: )</p>
<p>I just changed my github profile name.</p>
<p>In school we made a team project : <a href="https://github.com/simonbruant/projet_afpa_car/graphs/contributors" rel="nofollow noopener">https://github.com/simonbruant/projet_afpa_car/graphs/contributors</a></p>
<p>I appear in the project with my old name “Demky” but when i hover the name it show my actual account.</p>
<p>if I click the name it redirect me to a page url with the old name (it 404) : <a href="https://github.com/Demky" rel="nofollow noopener">https://github.com/Demky</a></p>
<p><img src="https://user-images.githubusercontent.com/37147299/181070032-02966e4e-3bd6-4b27-a064-1d9966eb6e0b.png" alt="git_name.png" data-base62-sha1="b6CHrU1R9ALWGeBJcwUrhPl7j81" width="424" height="409"></p>
<p><strong>is there is something I can do or should i change back my name to it’s original ?</strong></p>
<p>it’s sad, GitHub is able to track my new profile so why can’t it change the url too ?</p>
<p>from <strong>demky</strong> to <a href="https://github.com/" rel="nofollow noopener">https://github.com/</a> <strong>borisBelloc</strong></p>
<p>On my profile i didnt lost any “contribution” (green square).</p>
<p>thanks for help</p>

**Comment by borisBelloc** [comment, marked as answer]
<p>Ok, I did nothing and today, it fixed itself :slight_smile: Cool.</p>
<p>Look like GitHub just needed time (+/-24hours)</p>
<p><img src="https://user-images.githubusercontent.com/37147299/181070034-530e5cc5-b7fc-4a47-9c2f-5a4f1d51de7f.png" alt="a.PNG" data-base62-sha1="pyZ9tF5rccdVbmpp9dBOEq5t9zu" width="394" height="217"></p>

**Comment by AndreaGriffiths11** [comment]
<p>Hey <a class="mention" href="/borisbelloc">@borisbelloc</a>,</p>
<p>Thanks a lot for coming back to share that is fixed!</p>

**Comment by majormoses** [comment]
<p>There is some delay from when the redirects are put in place and analytics are re-run. That being said I often see commits where there is no email associated with the user. If you have write access to the repository it’s possible to change the author email address so that it is associated with one under their account like so: <a href="https://stackoverflow.com/questions/3042437/how-to-change-the-commit-author-for-one-specific-commit" rel="nofollow noopener">https://stackoverflow.com/questions/3042437/how-to-change-the-commit-author-for-one-specific-commit</a> that being said it’s generally poor form to do a rebase on the shared/public repo as it makes it hard for people to re-integrate back in without nuking their local copy. This is one of the reasons that some OSS projects have you make pull requests against a particular fork which then the maintainer will then integrate your changes into the main repository after cleaning up history.</p>
